suite/tests/api/drdecode_x86.c

```diff
@@ -155,6 +155,57 @@ test_noalloc(void)
      */
 }
 
+static void
+test_categories(void)
+{
+    const uint raw[] = {
+        0x10100ff3, /* fp move avx; OP_vmovss; movss  xmm2,DWORD PTR [eax] */
```

Comment:
I would suggest synthesizing these from DR IR and encoding them and then decoding from those encoded raw bits.  Then no human needs to deal with raw bits; we don't need these comments since we can read the IR fairly well; no worries about >4-byte encodings; easier to tweak opcodes or operands; easier to add new entries; etc.  Maybe the same thing for the aarch64 test.